### PR TITLE
Body JSONPath matcher

### PR DESCRIPTION
**Pull request by undera**

### CODE REVIEW

File: mockintosh/handlers.py [hunk 1]
```diff
@@ -660,10 +665,19 @@ def __init__(self):
...
+        self._json = NotParsedJSON()
         self.files = {}
         self.formData = {}
 
+    @property
+    def json(self):
+        if isinstance(self._json, NotParsedJSON):
+            try:
+                self._json = json.loads(self.body)
+            except json.JSONDecodeError:
```
Comment (undera) [inline mockintosh/handlers.py:677]: How will user know about the problem? No logging, no messages passed anywhere... That will be a confusing product

**Review by undera — COMMENTED** [review]
Dropping coverage for more than 1% is not good. Let's keep it covered.
Also please address my comment.

File: mockintosh/handlers.py [hunk 2]
```diff
@@ -660,10 +665,20 @@ def __init__(self):
         self.path = None
         self.headers = {}
         self.queryString = {}
-        self.body = None
```
Comment (undera) [inline mockintosh/handlers.py:663]: You have removed `body` attribute while code still uses it. Looks not right

File: mockintosh/handlers.py (hunk 1 above)
Comment (mertyildiran) [reply mockintosh/handlers.py:677]: Added logging with https://github.com/up9inc/mockintosh/commit/d7dcadaf71c26aa74e30782b060ad355980150b2

File: mockintosh/handlers.py (hunk 2 above)
Comment (mertyildiran) [reply mockintosh/handlers.py:663]: Fixed with https://github.com/up9inc/mockintosh/commit/55a7dfb21b8abb046fa10d675f90cf976c91e3f8

**Comment by mertyildiran** [comment]
> Dropping coverage for more than 1% is not good. Let's keep it covered.
> Also please address my comment.

@undera added tests for the implemented feature https://github.com/up9inc/mockintosh/commit/2d2872b9153b44523a6c5f0145ba8a38ba7aa0e0

**Pull request merged**